### PR TITLE
Feature/hooks

### LINKED ISSUE #23: [Device hook] Create hook to detect device
Implement hook which can detect mobile or desktop resolution of the screen.

For mobile screen should return "true".
For desktop screen should return "false".

Breakpoint between desktop/mobile resolution - see *SRS (design)

### Estimation: 1h

SRS - [SRS](https://docs.google.com/document/d/1Z0ceG_NtqwGXs6xRjqq8hb9AoqWj1lLzboL5epCSfkQ/edit#)

### PR DESCRIPTION
**Pull request by solomiia-sahala**
Added detect device hook

### CODE REVIEW

File: src/styles/variables.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
+//colors
```
Comment (alexandrbig) [inline src/styles/variables.scss]: seems like this file should not be in that PR, please remove it

File: public/index.html [hunk 2]
```diff
@@ -4,27 +4,11 @@
...
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
     <title>CoKeeper</title>
+    <link
```
Comment (alexandrbig) [inline public/index.html:8]: I saw this changes in different commit, why do we need it here?

File: src/styles/variables.scss (hunk 1 above)
Comment (solomiia-sahala) [reply src/styles/variables.scss]: removed this file.

File: public/index.html (hunk 2 above)
Comment (solomiia-sahala) [reply public/index.html:8]: yes, there is other request with this changed, but accidentally they appeared here as well. I will investigate how come.

**Review by alexandrbig — APPROVED** [review]

**Pull request merged**